main.tf

```diff
@@ -37,3 +38,19 @@ resource "github_organization_project" "project" {
   name = each.value.name
   body = each.value.body
 }
+
+resource "github_team" "all" {
+  count = var.create_all_members_team ? 1 : 0
+
+  name        = var.all_members_team_name
+  description = "This teams contains all members of our organization."
+  privacy     = "closed"
```

Comment:
should be dynamic, but can be added in a later version

Reply:
resolved in bbfe8d0